app/mailers/user_mailer.rb

```diff
@@ -9,4 +9,13 @@ def rent_created(rent_id)
          subject: default_i18n_subject,
          date: Time.zone.now)
   end
+
+  def rent_expired(rent_id)
+    @rent = Rent.find(rent_id)
+    @book = @rent.book
+    @date = Time.zone.now
+    mail(to: @rent.user.email,
+         subject: 'Rent expired',
```

Comment:
Why not using I18n for the subject?

Reply:
I didn't have one created and I forgot to add it later. I'll do it now.